recipes/dtscalibration/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "dtscalibration" %}
+{% set version = "3.1.0" %}
+{% set python_min = "3.9" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```